docs/design/cert-rotation.md

```diff
@@ -0,0 +1,64 @@
+# Cluster Certificate Rotation Feature Design
+
+Design doc on the certificate rotation feature for AKS Engine clusters.
+
+## Problem
+
+Until PR [#396](https://github.com/Azure/aks-engine/pull/396), the default expiration of the certs generated by AKS Engine was 2 years. Clusters that were deployed with an AKS Engine version prior to that PR will soon reach expiration. We currently do not have an outlined process for rotating certificates in an existing cluster.
+
+## Goals
+
+Provide a process that is easy to follow for rotating CA, etcd, kubelet, kubeconfig and apiserver certificates in a cluster built with AKS Engine.
+
+## Non-Goals
+
+- Providing a certificate rotation tool for AKS clusters
```

Comment:
Full stops here, please.

Reply:
you mean periods?

Reply:
nevermind, it became clear in the following comments :)

Reply:
Yeah. I try to refrain from AmE as much as possible :P.